assets/svg/graphics/cta-graph-arrow-up.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="6" height="6" viewBox="0 0 6 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 3.00921L0.442775 3.39663L2.19817 1.86341V5.20738H2.82622V1.86341L4.57848 3.39938L5.02439 3.00921L2.5122 0.811035L0 3.00921Z" fill="#C7C7C7"/>
```

Comment:
@techanvil fill must be set to `currentColor` instead of the hardcoded `#c7c7c7`. Thus we can set the color via CSS.

Reply:
Thanks, that's a great idea. I'll make the change.

Let me know if you think we should do the same for the other two SVGs. My thinking is they are "graphics" as opposed to "icons" and unlikely to be used in another context, so can keep their inline colors. However I would happy to change this too.

Reply:
@techanvil You're correct in your assumption. Graphics are not meant to be changed via CSS and can keep their colors 👍